docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -532,8 +532,8 @@ The default is 127.0.0.1.
 p2p-port="1789"
 ```
 
-Specifies the port on which P2P peer discovery listens.
-The default is 30303.
+Specifies the P2P peer discovery listening port (UDP and TCP).
```

Comment:
```suggestion
Specifies the P2P listening ports (UDP and TCP).
```

Reply:
To clarify - this doesn't just set the discovery listening port, so I think its more accurate to leave out "peer discovery".

Reply:
Updated